frontend/src/lib/router.tsx

```diff
@@ -114,6 +115,15 @@ const defaultRoutes: {
       </Chooser>
     ),
   },
+  dockerDesktopSetup: {
+    path: '/docker-desktop-setup',
+    exact: true,
+    name: 'Docker Desktop Setup',
+    sidebar: null,
+    useClusterURL: false,
+    noAuthRequired: true,
+    component: () => <DockerDesktopSetup />,
+  },
```

Comment:
I'd have preferred a link in order to save us time in the future, but since you already spent some time making this look good and since the UX is probably better, we can keep it, but please add this route only when running in DD.